third_party/islet/setup.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -e
```

Comment:
This is good, but over time, ... I have been taught (& discovered) that a more stricter checking for errors during shell script execution is better.

See if you can expand this to `set -Eeuo pipefail`, so that any errors in any command will cause this script to abort immediately.

This business of `$rc` error checking in bash scripts is bit difficult to work with. I experimented with some of this most recently in above-mentioned shell script. You may want to review what I did, to see if that model will work for your script. And adapt that accordingly.

Reply:
Also it is a good idea to always run `shellcheck` on this script, and fix all warnings / errors reported. Have you done that?

Reply:
I didn't use `shellcheck`. (I actually have used `shfmt` on my projects).
Thanks for valuable comments! I reflected both comments.